src/game/pointers/Pointers.hpp

```diff
@@ -1,19 +1,43 @@
 #pragma once
 #include <windows.h>
-#include <dxgi1_2.h>
+#include <dxgi1_4.h>
+#include <d3d12.h>
 
+//Someone can correct this
+enum class eRenderingType : uint8_t {
+	DX12 = 1,
+	Unk,
+	Vulkan
+};
+class RenderingInfo {
+public:
+	virtual ~RenderingInfo() = default;
+
+	uint8_t unk_0008; //0x0008
+	eRenderingType m_rendering_type; //0x0009
+
+	bool is_rendering_type(eRenderingType type) {
+		return m_rendering_type == type;
+	}
+	void set_rendering_type(eRenderingType type) {
+		if (!is_rendering_type(type)) {
+			m_rendering_type = type;
+		}
+	}
+};
```

Comment:
Move this outside of the pointers class.

Perhaps somewhere under `src/game/rdr/`?